group_vars/all.yml

```diff
@@ -68,6 +68,9 @@ eth_provision: "eth2"
 # ethernet interface on computing nodes( for non-warewulf environment )
 cnode_eth_internal: "eth2"
 
+# GNU Toolchain version to use:
+gnu_version: "8"
```

Comment:
This would force us to use gnu and we need to start looking into llvm.

A "toolchain" variable, set to "gnu8" for now would be enough, as I think packages would repeat the same pattern.